nautobot/extras/jobs.py

```diff
@@ -769,8 +768,7 @@ def save_file(uploaded_file):
         fp = FileProxy.objects.create(name=uploaded_file.name, file=uploaded_file)
         return fp.pk
 
-    @staticmethod
```

Comment:
Why no longer a static method?

Reply:
I needed access to the object to be able to call `self.logger`